wire-ios-sync-engine/Source/SessionManager/SessionManager.swift

```diff
@@ -1044,33 +1045,117 @@ public final class SessionManager: NSObject, SessionManagerType {
             onStartMigration: { [weak self] in
                 self?.delegate?.sessionManagerWillMigrateAccount(userSessionCanBeTornDown: {})
 
-            }, onFailure: { [weak self] error in
+            },
+            onFailure: { [weak self] error in
                 self?.delegate?.sessionManagerDidFailToLoadDatabase(error: error)
                 onCompletion(nil)
-
-            }, onCompletion: { [weak self] coreDataStack in
+                
+            },
+            onCompletion: { [weak self] coreDataStack in
                 guard let self else {
                     assertionFailure("expected 'self' to continue!")
                     return
                 }
 
-                let userSession = startBackgroundSession(
-                    for: account,
-                    with: coreDataStack
+                let journal = try! JournalStore(
```

Comment:
I'm not sure what to do with these possible errors. It's hard to bubble errors up, but we also can't go forward without the journal.

Reply:
this is now irrelevant with the new `Journal`.